packages/sdks/snippets/angular-ssr/src/app/blog-article/blog-article.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { BlogArticleComponent } from './blog-article.component';
+
+describe('BlogArticleComponent', () => {
+  let component: BlogArticleComponent;
+  let fixture: ComponentFixture<BlogArticleComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [BlogArticleComponent]
+    });
+    fixture = TestBed.createComponent(BlogArticleComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
I wouldn't add specs like these: all they do is check that the component rendered. it's unnecessary: we already have a comprehensive integration test checking the behaviour of the rendered page, which is more useful and applies to every snippet. This one only exists in one snippet (it's also not running in CI)

Reply:
@samijaber My bad, I forgot to remove these boilerplate files generated by CLI. I didn't add any test cases there. I will remove it and push it again.